src/app/integrations/create-page/select-connection/select-connection.component.html

```diff
@@ -1,3 +1,3 @@
 <p>Choose a connection to start</p>
-<ipaas-connections-list-toolbar></ipaas-connections-list-toolbar>
-<ipaas-connections-list [connections]="connections | async" [loading]="loading | async" (onSelected)="onSelected($event)"></ipaas-connections-list>
\ No newline at end of file
+<ipaas-connections-list-toolbar  [(filter)]="filter" [(sort)]="sort"></ipaas-connections-list-toolbar>
+<ipaas-connections-list [connections]="connections | async | objectPropertyFilter:filter | objectPropertySort:sort" [loading]="loading | async" (onSelected)="onSelected($event)"></ipaas-connections-list>
```

Comment:
`yarn lint` let you get away with ending a file without a newline? I thought we had `eofline` enabled ([ref](https://palantir.github.io/tslint/rules/eofline/)) [here](https://github.com/redhat-ipaas/ipaas-ui/blob/master/tslint.json#L12). @jimmidyson to the rescue